class.jetpack.php

```diff
@@ -7124,4 +7133,61 @@ function is_active_and_not_development_mode( $maybe ) {
 		}
 		return true;
 	}
+
+	/* Activated module */
+	function track_activate_module( $module ) {
+		$this->tracking->record_user_event( 'module_activated', array( 'module' => $module ) );
+	}
+
+	/* Deactivated module */
+	function track_deactivate_module( $module ) {
+		$this->tracking->record_user_event( 'module_deactivated', array( 'module' => $module ) );
+	}
+
+	/* User has linked their account */
+	function track_user_linked() {
+		$user_id = get_current_user_id();
+		$anon_id = get_user_meta( $user_id, 'jetpack_tracks_anon_id', true );
+
+		if ( $anon_id ) {
+			self::record_user_event( '_aliasUser', array( 'anonId' => $anon_id ) );
+			delete_user_meta( $user_id, 'jetpack_tracks_anon_id' );
+			if ( ! headers_sent() ) {
+				setcookie( 'tk_ai', 'expired', time() - 1000 );
+			}
+		}
+
+		$wpcom_user_data = Jetpack::get_connected_user_data( $user_id );
```

Comment:
Ideally, we'd like to use the connection package for this, but for now we could also use `self`:

```suggestion
		$wpcom_user_data = self::get_connected_user_data( $user_id );
```